fr/releases/centreon-commercial-extensions.md

```diff
@@ -18,6 +18,14 @@ commerciales, veuillez contacter le support.
 
 ## Centreon MAP release notes
 
+### 20.04.4
+
+* Sauvegarder une Geo View sur un Widget Custom View MAP était impossible dû à l'absence du bouton "Sauvegarder"
+* Partager les métriques de statistiques d'usage entre le serveur MAP et le Central en HTTPS échouait
+* Créer des Liens en utilisant l'API REST peut maintenant recourir au propriétés 'bendpoints', 'displayValue' et 'displayPercent'
+* Ajouter des ressources Centreon lors de la création d'un Widget Output par l'API REST est maintenant possible
+* Installer MAP Studio lors d'une fress install échouait à configurer le back-up automatique
```

Comment:
Was it because of the automatic backup that we couldn't install the desktop client? 

```suggestion
* Installer MAP Studio lors d'une nouvelle installation échouait à cause de la configuration du back-up automatique
```

Reply:
From what I understand, it did not prevent installing anything, just prevented back-up from working because of default file path